app-drawer/app-drawer.html

```diff
@@ -535,6 +538,39 @@
         this.fire('app-drawer-overlay-changed', { overlay: this.opened && !this.persistent });
       },
 
+      _openedChanged: function(opened) {
+        var evtName = this.opened ? 'opened' : 'closed';
+
+        this.fire('app-drawer-' + evtName);
+      },
+
+      _onTransitionEnd: function(evt) {
+        var opened = this.opened;
+        var evtName = this.opened ? 'opened' : 'closed';
+
+        this.fire('app-drawer-transitioned-' + evtName);
```

Comment:
Similarly, rely on `app-drawer-overlay-changed` (which handles de-duplicating `transitionend` events, since more than one child is transitioned - see https://github.com/PolymerLabs/app-layout/blob/master/app-drawer/app-drawer.html#L556), then read the `opened` property